docs/modules/tiered-storage/pages/capacity_planning.adoc

```diff
@@ -0,0 +1,46 @@
+= Capacity Planning
```

Comment:
Do we need some recommendations and examples here?

Reply:
We already composed an example out of the recommendations for memory capacity.
Maybe a similar example can be given for disk capacity. WDYT @taburet , @ethurma , @blazember ?

Reply:
Sorry for being a bit long here, I'm adding some explanation.

We could be specific about memory capacity, because we keep data structures in the memory with there memory footprint depending on the record count, and there is a minimum defined by the dimensions of the member (partition count, page size). For the disk capacity we don't have similar fix costs we can calculate with.

What we can tell about the disk capacity is that it should be `total_data_size - effective_memory_capacity + disk_overhead`, where `effective_memory_capacity` is 1) vague; and we'd need to go into details that are hard to explain, and 2) it's typically not the dominant component with the typical data volumes targeted by TS, so I wouldn't use this formula. 

What we can say is that size should be like `total_data_size + overhead`, where the `overhead` is a function of the disk-space vs compaction performance trade-off the user intends to make. After all, what we can say is that
```
disk_capacity = total_data_size + partition_count * accepted_garbage_per_partition_in_mb * 10
```
is a safe recommendation that won't choke the cluster by the too frequent compactions. `accepted_garbage_per_partition_in_mb` is what `hazelcast.tiered.store.partition.compactor.gc.threshold.per.map.partition.in.mb` defines. So with default parameters, and for 100GB total data size our formula is
```
disk_capacity = 100GB + 271*8MB*10 -> ~121GB
```
We can also mention that the overhead is a function of the settings, and not that of the data volume. The formula with 1TB data volume gives us 1021GB, so ~20% overhead could be an unreasonably high value.

Note that `total_data_size` is the data volume in the cluster, not the member's share of that. The reason is that we may have splits etc where all data ends up in a single member.

@petrpleshachkov @ahmetmircik do you have any additional thoughts on this?

Reply:
it is ok with me

Reply:
Addressed in https://github.com/hazelcast/hz-docs/pull/1047/commits/bc45d7991964c8bbd32c1f990e6f8103e5567e71